runtime/lua/vscode-neovim/internal.lua

```
@@ -268,4 +268,15 @@ function M.start_visual(buf, anchor, active)
   api.nvim_feedkeys((v == "V" or v == "\x16") and "gvv" or "gv", "n", false)
 end
 
+---Translate from a Windows path to a WSL path
+---@param path string
+---@return string?
+function M.wslpath(path)
+  local f = io.popen(("wslpath '%s'"):format(path))
+  if f then
```

wouldn't you want an error instead of silently ignoring a failure?